war/src/main/resources/templates/scores/details.html

```diff
@@ -110,6 +113,24 @@ <h3>Probes results</h3>
             <tbody>
             <tr data-th-each="item: ${plugin.details}" data-th-with="probeResult = ${item.value}">
                 <td><code data-th-text="${probeResult.id}" class="label"></code></td>
+                <td>
+                    <ion-icon data-th-if="${probeResult.status.name} == 'SUCCESS'" class="success" name="success"></ion-icon>
+                    <ion-icon data-th-unless="${probeResult.status.name} == 'SUCCESS'" class="failure" name="failure"></ion-icon>
+                </td>
```

Comment:
That was removed in #501. Don't add it again.

Reply:
Changed @alecharp